README.md

```diff
@@ -35,7 +35,7 @@ Need more details? See the [Faraday API Documentation][apidoc] to see how it wor
 This library aims to support and is [tested against][actions] the currently officially supported Ruby
 implementations. This means that, even without a major release, we could add or drop support for Ruby versions,
 following their [EOL](https://endoflife.date/ruby).
-Currently that means we support Ruby 2.6+
+Currently that means we support Ruby 3.0+
```

Comment:
You've mentioned some "outdated docs" below. Do we need to correct it?

Reply:
This was the outdated part I was referring to in the discussion thread. Ruby 2.6 has been EOL for quite some time, but it was still referred to here anyway 😅